pkg/controller/registry/resolver/resolver.go

```diff
@@ -397,6 +408,14 @@ func (r *SatResolver) newSnapshotForNamespace(namespace string, subs []*v1alpha1
 		installables = append(installables, &i)
 	}
 
+	if len(csvsMissingProperties) > 0 {
+		names := make([]string, len(csvsMissingProperties))
+		for i, csv := range csvsMissingProperties {
+			names[i] = csv.GetName()
+		}
+		r.log.Infof("considered csvs without properties annotation during resolution: %v", names)
```

Comment:
I'm wondering if it makes more sense to emit this log per CSV, so its more human readable.

For example, instead of 
`considered csvs without properties annotation during resolution: CSVA, CSVB, CSVC` 
we have 
`considered CSVA without properties annotation during resolution` 
`considered CSVB without properties annotation during resolution` 
`considered CSVC without properties annotation during resolution` 

WDYT?

Reply:
I am worried about the noise level. Under this change, it would be normal/harmless for users on pre-existing clusters to see this message on every resolution cycle. What might eventually make sense is to persist the inferred annotations up-front, so there would be a single message per inferred CSV. I just didn't want to do that until I was confident that there was nothing else we wanted to infer from an unannotated CSV.

Reply:
ok, makes sense